Bottom: d74da065e18a76e2f13bdc3ac76e7f3a6fd4ecfb
Top:    cafbff9b377b9ad92c2d6fc854c1bf23d9036290
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 01:49:01 +0100

Refresh of data-c-remove-unused-global-th

---

diff --git a/src/data.c b/src/data.c
index 412b305..4ebc126 100644
--- a/src/data.c
+++ b/src/data.c
@@ -63,8 +63,6 @@ int addedLines;
 
 LocationType vxl, vxh, vyl, vyh;	/* visible pcb coordinates */
 
-BoxType clipBox;		/* box for clipping of drawing */
-
 double Zoom_Multiplier = 0.01;
 
 /*  { 1.5625, 2.2097, 3.125, 4.4194, 6.25, 8.8388,
diff --git a/src/data.h b/src/data.h
index fe98735..6e70fd3 100644
--- a/src/data.h
+++ b/src/data.h
@@ -61,8 +61,6 @@ extern LibraryType Library;
 
 extern LocationType Xorig, Yorig;
 
-extern BoxType clipBox;
-
 extern double Zoom_Multiplier;
 
 extern char *Progname;
